app/utils/countrycodes.py

```diff
@@ -374,4 +374,11 @@ def country_code(country):
         country = synonyms[country]
 
     # Get country or fallback to default_code.
-    return is_3166_1.get(country, default_code)
+    country_code = is_3166_1.get(country, default_code)
+
+    # Default picked?
+    if country_code == default_code:
+        print("No country_code found for '%s'. Using '%s'!" % (country, country_code))
```

Comment:
We should use the logger.
```python
import logging

LOGGER = logging.getLogger(__name__)

...

# Default picked?
    if country_code == default_code:
        LOGGER.warning(f"No country_code found for '{country}'. Using '{country_code}'!")
```
Also switched the formatting syntax to use fstrings.
https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python
https://realpython.com/python-string-formatting/


```

Reply:
C'mon guys do we really explain that "return" means "return"?
```
    # Return.
    return country_code
```

Although, if "Brexit means Brexit"... Ok, ok, maybe you're right. I'll give it a thought :)

Reply:
Alright, changed. Btw, the ``isort`` and ``black`` are still conflicting I believe. Had to manually change ``test_routes`` and ``conftest`` back after black modified it.